jupytercalpoly/Jupyter-Bifrost
# Customize mark of the graph

## Changes
* Added the three subtabs(mark, scale, style) in customization tab.
* Added mark options that users can select in mark subtab based on the current x and y variables.
![Kapture 2021-07-08 at 17 36 30](https://user-images.githubusercontent.com/15991814/125006568-4f0a2f00-e013-11eb-9f3b-101876fe5d64.gif)
![Kapture 2021-07-12 at 16 14 43](https://user-images.githubusercontent.com/15991814/125367230-a1599180-e32c-11eb-9f4b-34dd21a97911.gif)
* Fixed the bifrost widget styling affecting the jupyterlab ui (button, text)
* Fixed the error when users give zero or one variable for the graph.
    * When the zero variable is given, it doesn't show any graph.
    * When one variable is given, it only suggests all possible charts but pie chart.

## Testing
* Plot a graph with bifrost.
* Click on the customization tab on sidebar.
* Click on mark subtab on the customization tab.
* Select one of the charts.
* See changes in the main graph.

## Future Work
* Transition from a plot to a plot with a different encoding
